src/robusta/integrations/prometheus/models.py

```diff
@@ -91,6 +92,13 @@ def get_title(self) -> str:
         else:
             return self.alert.labels.get("alertname", "")
 
+    def get_prometheus_query(self) -> str:
+        """
+        Gets the prometheus query that defines this alert.
+        """
+        url = urlparse(self.alert.generatorURL)
+        return re.match(r"g0.expr=(.*)&g0.tab=1", unquote_plus(url.query)).group(1)
```

Comment:
In case the prometheus alert query isn't what you expect, wouldn't you want to send the alerts without that enrichment?
I think re.match() can return None

Reply:
I don't think it's possible though. If this occurs we want to know somehow so we can fix the bug.

Can we add protection here as a general mechanism that if one playbook action fails we still run the other playbook actions?